client/src/component/searchImage.jsx

```diff
@@ -9,6 +9,7 @@ class searchImage extends Component {
         super(props);
         this.state = {
             images: [{src: 'client/src/InitImg.jpg', width: 1, height: 1}],
+            moreImage: [{src: 'client/src/InitImg.jpg', width: 1, height: 1}],
```

Comment:
Can you explain why you need a separate variable for this?

Reply:
to store images loaded in background

when scroll is down 50% images stored in that variables are appended to displaying gallery

Reply:
change name to nextImages